06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/Output/app.js

```diff
@@ -0,0 +1,126 @@
+/* global angular, InvertedIndex, document, toastr*/
```

Comment:
Please ensure that your variable/parameter names are not abbreviated and are in camelCase for this entire file.